src/components/aside/index.ts

```diff
@@ -0,0 +1,21 @@
+import Handlebars from 'handlebars';
+import template from './template';
+import Block from '../../modules/Block';
+
+import './styles.module.css';
+
+interface Props {
+  class?: string;
+  contentTop: Block | Block[];
+  content: Block | Block[];
+}
+
+export default class Aside extends Block {
+  constructor(props: Props) {
+    super('div', props);
+  }
+
+  render(): Function {
+		return Handlebars.compile(template, { noEscape: true });
+	}
```

Comment:
```suggestion
  render(): Function {
    return Handlebars.compile(template, { noEscape: true });
  }
```
А так же лучше возвращать отсюда не функцию, а ее результат (строку). Тогда будет возможность править контекст компиляции из компонента, а поскольку ты компилишь ее в `Block`, то у тебя не будет возможности кастомизировать/модифицировать контекст

Reply:
Но тогда, если я не ошибаюсь, нельзя будет навешивать обработчики

Reply:
Да нет же, просто место компиляции изменится, обработчики как вешались после компиляции шаблона, так и будут

Reply:
то есть будет что-то в виде 
`this.compile(template, this.props)`
?

Reply:
```
const template = `
  <button prop={{prop}}>{{additionalProp}}</span>
`
const compile = Handlebars.compile(template);

class Button extends Block {
  render() {
    return compile({
      ...this.props,
      additionalProp: 'foo',
      modificatedProp: this.props.prop + 'bar',
      ...
    })
  }
}
```
Только в Block нужно учесть что будет приходить уже строка